src/Observer/ConvertQuoteToOrder.php

```diff
@@ -25,19 +34,26 @@ public function execute(Observer $observer)
         $order = $observer->getOrder();
 
         $methodCode = $order->getShippingMethod();
-        if (str_starts_with($methodCode, 'inpostlocker_')) {
-            $lockerId = $quote->getInpostLockerId();
-            $pickupDesc = $this->session->getData('inpost_locker_address_' . $lockerId);
-
-            $description = implode(
-                PHP_EOL,
-                [
-                    $order->getShippingDescription(),
-                    $pickupDesc,
-                ]
-            );
-
-            $order->setShippingDescription($description);
+        if (!str_starts_with($methodCode, 'inpostlocker_')) {
+            return;
+        }
+
+        $lockerId = $quote->getInpostLockerId();
+        if (empty($lockerId)) {
+            $this->logger->critical('snow_inpost', ['quote_id' => $quote->getId()]);
+            throw new LocalizedException(__('Choose locker again'));
```

Comment:
Code should never enter this, but if that's the case then customers will be forced to re-select their paczkomat